src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/ExtensionMethodImportCompletionHelper.cs

```diff
@@ -95,7 +101,8 @@ internal static partial class ExtensionMethodImportCompletionHelper
             var getSymbolsTicks = Environment.TickCount - ticks;
             ticks = Environment.TickCount;
 
-            var items = ConvertSymbolsToCompletionItems(extentsionMethodSymbols, cancellationToken);
+            var compilation = await document.Project.GetRequiredCompilationAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
Wondering if we could reuse some existing compilation? Like the one with the SyntaxContext?

Reply:
What do you mean "the one with the syntax context"? No SyntaxContext is created or passed in

Reply:
I mean here.
http://sourceroslyn.io/#Microsoft.CodeAnalysis.Workspaces/SyntaxContext.cs,76
Could it use the SemanticModel.Compilation in SyntaxContext? 
I think this method is called somewhere when we try to provide CompletionItem, so SyntaxContext should be available somewhere.

Reply:
it looks like this method is called from here
http://sourceroslyn.io/#Microsoft.CodeAnalysis.Features/Completion/Providers/ImportCompletionProvider/AbstractExtensionMethodImportCompletionProvider.cs,28

Could we pass the syntaxContext.SenmanticModel.Compilation to this method and reuse that?

Reply:
OK I see what you meant now. The answer is we can't, since the work actually happens in remote process. i.e. `ExtensionMethodImportCompletionProvider` run in devenv and calls `ExtensionMethodImportCompletionHelper.GetUnimportedExtensionMethodsAsync`, which in turn calls `IRemoteExtensionMethodImportCompletionService.GetUnimportedExtensionMethodsAsync`.